yt_dlp/extractor/ntvde.py

```diff
@@ -33,41 +34,38 @@ def _real_extract(self, url):
         video_id = self._match_id(url)
         webpage = self._download_webpage(url, video_id)
 
-        info = self._parse_json(self._search_regex(
-            r'(?s)ntv\.pageInfo\.article\s*=\s*(\{.*?\});', webpage, 'info'),
-            video_id, transform_source=js_to_json)
-        timestamp = int_or_none(info.get('publishedDateAsUnixTimeStamp'))
-        vdata = self._parse_json(self._search_regex(
-            r'(?s)\$\(\s*"\#player"\s*\)\s*\.data\(\s*"player",\s*(\{.*?\})\);',
-            webpage, 'player data'), video_id,
-            transform_source=lambda s: js_to_json(re.sub(r'advertising:\s*{[^}]+},', '', s)))
-        duration = parse_duration(vdata.get('duration'))
+        info = self._search_json(
+            r'article:\s*', webpage, 'info', video_id, transform_source=js_to_json)
+
+        player_data = self._search_json(
+            r'\$\(\s*"#playerwrapper"\s*\)\s*\.data\(\s*"player",\s*',
+            webpage, 'player data', video_id,
+            transform_source=lambda s: js_to_json(re.sub(r'ivw:[^},]+', '', s)))
+        vdata = traverse_obj(player_data, ('setup', 'source')) or {}
```

Comment:
If we really want `vdata`-related code to be non-fatal, then we'd want this:
```suggestion
        vdata = traverse_obj(self._search_json(
            r'\$\(\s*"#playerwrapper"\s*\)\s*\.data\(\s*"player",',
            webpage, 'player data', video_id, default={},
            transform_source=lambda s: js_to_json(re.sub(r'ivw:[^},]+', '', s))),
            ('setup', 'source', {dict})) or {}
```

&nbsp;

But do we actually want it to be non-fatal? We don't have a video without it. This would be fine imo:
```suggestion
        vdata = self._search_json(
            r'\$\(\s*"#playerwrapper"\s*\)\s*\.data\(\s*"player",',
            webpage, 'player data', video_id,
            transform_source=lambda s: js_to_json(re.sub(r'ivw:[^},]+', '', s)))['setup']['source']
```